app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+
+/**
+ * 유저 정보를 관리하는 서비스.
+ */
+public class UserService {
+    /**
+     * 주어진 유저를 생성합니다.
+     *
+     * @param user 생성할 정보가 담긴 유저 객체.
```

Comment:
`@param` 태그에는 마침표를 찍지 않습니다.
```suggestion
     * @param user 생성할 정보가 담긴 유저 객체
```

Reply:
아! 감사합니다!